### PR TITLE
Created widgets for each tab + added tab switching logic. Also created a journal entry input form.

### PR DESCRIPTION
**Pull request by SheinH**

### CODE REVIEW

File: lib/main_tab_widget.dart [hunk 1]
```diff
@@ -1,5 +1,10 @@
 import 'package:flutter/cupertino.dart';
 import 'package:flutter/material.dart';
+import 'package:scbsss/models/journal_entry.dart';
```
Comment (CS9490) [inline lib/main_tab_widget.dart:3]: Not quite sure why but based on the file organization we have in our repo, this doesn't work... I changed it in my branch but from what I know it should be: "import 'models/main_tab_widget.dart';". At least that's what worked on my machine... Not sure why I need to create a different path. But regardless, we can make a fix for this

File: lib/models/journal_entry.dart
Comment (CS9490) [inline lib/models/journal_entry.dart:1]: This is a good start. We still have a few tables to add in but we can work around this for this particular feature.

**Review by CS9490 — APPROVED** [review]
It's clear to go on my end. We still need to put in the tables and we have a little bit more work to do for the first 2 week sprint we have. My only minor issue, which is likely personal, is the imports in the main.dart file don't really match up with what we have in the repository, but again this is a minor issue and is likely just my machine acting a bit funky. Overall, good progress! :D

EDIT: Can confirm the imports issue was just a me problem. It works just fine now on my end.

File: lib/main_tab_widget.dart (hunk 1 above)
Comment (CS9490) [reply lib/main_tab_widget.dart:3]: Never mind! Just figured out the issue. The file works just fine with these imports.

**Comment by CS9490** [comment]
The features you set up work just fine on my end. Of course there's still work to be done, but this is good to merge on my end. We can build upon it.

**Comment by armyf35** [comment]
Fixed the permissions on main so you should be able to merge into main now

**Pull request merged**